src/uct/ib/base/ib_device.h

```diff
@@ -72,16 +72,19 @@ typedef enum uct_ib_roce_version {
 
 
 enum {
-    UCT_IB_DEVICE_FLAG_MLX4_PRM = UCS_BIT(1),   /* Device supports mlx4 PRM */
-    UCT_IB_DEVICE_FLAG_MLX5_PRM = UCS_BIT(2),   /* Device supports mlx5 PRM */
-    UCT_IB_DEVICE_FLAG_MELLANOX = UCS_BIT(3),   /* Mellanox device */
-    UCT_IB_DEVICE_FLAG_LINK_IB  = UCS_BIT(5),   /* Require only IB */
-    UCT_IB_DEVICE_FLAG_DC_V1    = UCS_BIT(6),   /* Device supports DC ver 1 */
-    UCT_IB_DEVICE_FLAG_DC_V2    = UCS_BIT(7),   /* Device supports DC ver 2 */
-    UCT_IB_DEVICE_FLAG_AV       = UCS_BIT(8),   /* Device supports compact AV */
-    UCT_IB_DEVICE_FLAG_DC       = UCT_IB_DEVICE_FLAG_DC_V1 |
-                                  UCT_IB_DEVICE_FLAG_DC_V2, /* Device supports DC */
-    UCT_IB_DEVICE_FLAG_ODP_IMPLICIT = UCS_BIT(9),
+    UCT_IB_DEVICE_FLAG_MLX4_PRM     = UCS_BIT(1),               /* Device supports mlx4 PRM */
+    UCT_IB_DEVICE_FLAG_MLX5_PRM     = UCS_BIT(2),               /* Device supports mlx5 PRM */
+    UCT_IB_DEVICE_FLAG_MELLANOX     = UCS_BIT(3),               /* Mellanox device */
+    UCT_IB_DEVICE_FLAG_LINK_IB      = UCS_BIT(5),               /* Require only IB */
+    UCT_IB_DEVICE_FLAG_DC_V1        = UCS_BIT(6),               /* Device supports DC ver 1 */
+    UCT_IB_DEVICE_FLAG_DC_V2        = UCS_BIT(7),               /* Device supports DC ver 2 */
+    UCT_IB_DEVICE_FLAG_AV           = UCS_BIT(8),               /* Device supports compact AV */
+    UCT_IB_DEVICE_FLAG_DC           = UCT_IB_DEVICE_FLAG_DC_V1 |
+                                      UCT_IB_DEVICE_FLAG_DC_V2, /* Device supports DC */
+    UCT_IB_DEVICE_FLAG_ODP_IMPLICIT = UCS_BIT(9),               /* Device supports implicit ODP */
+    UCT_IB_DEVICE_FLAG_OOO_SL_MASK  = UCS_BIT(10)               /* Device supports querying
```

Comment:
these flags are set by device model and we plan to remove them
need to use mlx5 md flag instead

Reply:
done